Packs/HPE_Service_Manager/Integrations/HPE_Service_Manager/HPE_Service_Manager.yml

```diff
@@ -163,6 +153,76 @@ script:
         };
     } // queryToObject takes query of format field1=value1&field2=value2 and return query object like: { "field1": "value1", "field2": "value2" }
 
+    function createRequest() {
+        var newRequest = {
+            [params.resourcename]: {
+                Category: args.category,
+                Purpose: [
+                    args.purpose
+                ],
+                Service: args.service,
+                Title: args.title,
+            }
+        };
+
+        if (args.callbackcontactname) {
+            newRequest[params.resourcename]['CallbackContactName'] = args.callbackcontactname;
+        }
+        if (args.contactname) {
+            newRequest[params.resourcename]['ContactName'] = args.contactname;
+        }
+        if (args.title) {
+            newRequest[params.resourcename]['Title'] = args.title;
+        }
+        if (args.purpose) {
+             newRequest[params.resourcename]['Purpose'] = args.purpose;
+        }
+
+        if (args.callbackcontactname == args.contactname) {
+            requested_for = args.contactname;
+        } else {
+            requested_for = args.callbackcontactname;
+        }
+        newRequest[params.resourcename]['cartItems'] = [{ 'ItemName': args.title , 'Quantity': '1', 'RequestedFor': requested_for }];
+
+        //throw JSON.stringify(newRequest)
+        var res = doPost('/'+params.resourcename, newRequest);
+        var parsedRes = JSON.parse(res);
+        if (parsedRes.ReturnCode !== 0) {
+            throw 'Failed to create request. Error Response: ' + res.Messages;
+        }
+
+        req = parsedRes['Messages'][2];
+        request = req.split('"');
+
+        var srHPSM = {};
+        var lstHPSM = [];
+
+        srHPSM['ID'] = request[1];
+        srHPSM['CallbackType'] = parsedRes[params.resourcename]['CallbackType'];
```

Comment:
I think `CallbackType` was not added as an entry to `parsedRes[params.resourcename]`.